WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -90,9 +89,9 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 				x => x.IsExpanded),
 
 				// Date
-				new PrivacyTextColumn<HistoryItemViewModelBase>(
```

Comment:
The PrivacyTextColumn was specifically used to improve history table performance, especially scrolling perf.